crates/uplc/src/machine/runtime.rs

```diff
@@ -371,8 +371,8 @@ impl<'a> Machine<'a> {
             }
             DefaultFunction::SliceByteString => {
                 let arg1 = runtime.args[0].unwrap_integer()?;
-                let arg2 = runtime.args[1].unwrap_integer()?;
-                let arg3 = runtime.args[2].unwrap_byte_string()?;
+                let arg2: &'a num::BigInt = runtime.args[1].unwrap_integer()?;
```

Comment:
We don't really need to explicitly type here, the compiler can handle the types for us. Especially since we may at some point change the library we're using to handle bigints.